From c7aa10ed896db0a6eb12a405144bb8b22af1d107 Mon Sep 17 00:00:00 2001
From: Darren Etheridge <detheridge@ti.com>
Date: Wed, 29 Nov 2023 10:57:26 -0600
Subject: [PATCH 2/2] j7 platforms:  disable SUPPORT_CPUCACHED_FWMEMCTX

As per IMG ticket 137446, set SUPPORT_CPUCACHED_FWMEMCTX to 0
after patch PP137446_CL6514287 is applied.

Signed-off-by: Darren Etheridge <detheridge@ti.com>
---
 build/linux/j721s2_linux/Makefile | 2 +-
 build/linux/j784s4_linux/Makefile | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/build/linux/j721s2_linux/Makefile b/build/linux/j721s2_linux/Makefile
index 2b0f2c3..22fbeb3 100644
--- a/build/linux/j721s2_linux/Makefile
+++ b/build/linux/j721s2_linux/Makefile
@@ -51,7 +51,7 @@ endif
 NO_HARDWARE := 0
 PVR_LDM_PLATFORM_PRE_REGISTERED := 1
 ENABLE_EMULATED_LARGE_TEXTURES := 1
-SUPPORT_CPUCACHED_FWMEMCTX := 1
+SUPPORT_CPUCACHED_FWMEMCTX := 0
 
 SUPPORT_NATIVE_FENCE_SYNC := 1
 
diff --git a/build/linux/j784s4_linux/Makefile b/build/linux/j784s4_linux/Makefile
index 2b0f2c3..22fbeb3 100644
--- a/build/linux/j784s4_linux/Makefile
+++ b/build/linux/j784s4_linux/Makefile
@@ -51,7 +51,7 @@ endif
 NO_HARDWARE := 0
 PVR_LDM_PLATFORM_PRE_REGISTERED := 1
 ENABLE_EMULATED_LARGE_TEXTURES := 1
-SUPPORT_CPUCACHED_FWMEMCTX := 1
+SUPPORT_CPUCACHED_FWMEMCTX := 0
 
 SUPPORT_NATIVE_FENCE_SYNC := 1
 
-- 
2.34.1

